src/components/line/ItemThermometerStations.vue

```diff
@@ -0,0 +1,159 @@
+  },
+})
+const stationActive = ref<Boolean>(false)
+const borderColor = ref(getColorLine('border', props.line, 600))
+
+let desserteSplit = props.desserte
+  .split(' ')
+  .filter((bus) => bus !== undefined && bus != '')
+const desserteTab = ref<BusNumber[]>(
+  desserteSplit.map((bus) => {
+    let new_bus: BusNumber = bus as BusNumber
+    return new_bus
+  })
+)
+
+var lineTab = ref<LineNumber[]>([])
+if (props.li_code != '') {
+  let li_code_split = props.li_code.split(' ')
+  lineTab.value = li_code_split.map((line) => {
+    let new_line: LineNumber = parseInt(line) as LineNumber
+    return new_line
+  })
+}
+
+function getClassCircle() {
+  let marginLeftNegative = '-ml-1'
+  let border2px = 'border-[2px]'
+  if (stationActive.value) {
+    return [
+      'min-w-[16px]',
+      'w-[16px]',
+      'h-[16px]',
+      'border-black',
+      marginLeftNegative,
+      border2px,
+    ]
+  }
+  if (props.index == 1 || props.is_last_elem) {
+    return [
+      'min-w-[15px]',
+      'w-[15px]',
+      'h-[15px]',
+      marginLeftNegative,
+      borderColor.value,
+      'border-[4px]',
+    ]
+  }
+  return ['min-w-[8px]', 'w-[8px]', 'h-[8px]', borderColor.value, border2px]
+}
+
+function getClassBeforeCircle() {
+  if (props.index == 1) {
+    return []
+  }
+  var classBefore = [
+    'before:absolute',
+    "before:content-['']",
+    'before:border-l-[2px]',
+    'before:' + borderColor.value,
+  ]
+  let ml = 'before:ml-[1px]'
+  let bottom = 'before:bottom-[65%]'
+  let height = 'before:h-[95%]'
+  if (stationActive.value) {
+    ml = 'before:ml-[5px]'
+    bottom = 'before:bottom-[75%]'
+    height = 'before:h-[70%]'
+  } else if (props.is_last_elem) {
+    ml = 'before:ml-[3px]'
+    height = 'before:h-[85%]'
+  }
+  return classBefore.concat([ml, bottom, height])
+}
+
+const classCircle = computed(() => {
+  var classObject = ['rounded-[50%]', 'mr-2', 'mb-3', 'mt-2']
+  classObject = classObject.concat(getClassCircle())
+  classObject = classObject.concat(getClassBeforeCircle())
+  return classObject
+})
+</script>
+
+<template>
+  <li
+    class="flex items-center mb-1 cursor-pointer"
+    :class="stationActive ? 'bg-[#F1F5F9]' : ''"
+    @mouseover="stationActive = true"
+    @mouseleave="stationActive = false"
+  >
+    <div :class="classCircle" />
+    <p class="mb-1 station-title">{{ name }}</p>
+    <div class="ml-auto mr-[5px]">
+      <div class="flex">
+        <template v-for="(line_connected, index) in lineTab" :key="index">
+          <IconLine :line="line_connected" :size="'m'" class="mb-1 ml-[1px]" />
+        </template>
+        <template v-for="(bus, index) in desserteTab" :key="index">
+          <IconBus :bus="bus" :size="'m'" class="mb-1 ml-[1px]" />
+        </template>
+        <template v-if="parking !== undefined && parking">
+          <IconParking :size="'m'" class="mb-1 ml-[1px]" />
+        </template>
+      </div>
+    </div>
+  </li>
+</template>
+
+<style scoped>
+.station-title {
```

Comment:
in general, we try to avoid using custom class if possible and use the style class from Tailwind. Can you change it to use Tailwind directly?